packages/cli/src/ui/commands/setupGithubCommand.ts

```diff
@@ -29,7 +29,7 @@ export const setupGithubCommand: SlashCommand = {
 
     // TODO(#5198): pin workflow versions for release controls
     const version = 'main';
```

Comment:
should we also change this to `v0`

in future work, can make this an input for the command